CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing Guidelines
+
+Thank you for considering contributing to this package. Please take a moment to review [Storyblok's general contributing guidelines](https://github.com/storyblok/.github/blob/main/CONTRIBUTING.md) if it's your first time to find info on submitting PR or issues.
+
+## Contributing to the code
+
+Find useful these guidelines and set of recommendations to help improving this library.
+
+### Setup
+
+This package relies on [pnpm workspaces](https://pnpm.io/workspaces) to manage dependencies. For instructions on how to install pnpm, please visit [pnpm.io](https://pnpm.io/installation).
+
+```bash
+pnpm install
+```
+
+### Build
+
+```bash
+pnpm build
+```
+
+### Development
+
+We strongly recommend using the `playgrounds` within the package to develop and manually test the package.
```

Comment:
One recommendation, I would remove the code notation on "playgrounds", if it's not an API or a command it creates a false expectation.

If you want to highlight the word, use italics or bold.